packages/ods/src/components/form-field/src/components/ods-form-field/ods-form-field.scss

```diff
@@ -0,0 +1,24 @@
+:host(.ods-form-field) {
+  display: inline-flex;
+  position: relative;
+  flex-direction: column;
+}
+
+.ods-form-field {
+  &__top-bar {
+    display: flex;
+    flex-direction: row;
```

Comment:
add a `column-gap` to ensure spacing if label is long enough to reach the second element